<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>968</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>871</width>
      <height>631</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget::pane { /* The tab widget frame */
	font: 8pt &quot;Leelawadee UI Semilight&quot;;
	font: 57 8pt &quot;Yu Gothic Medium&quot;;
	font: 8pt &quot;Times New Roman&quot;;
	font: 75 8pt &quot;MS Shell Dlg 2&quot;;
    border-top: 2px solid #C2C7CB;
}

QTabWidget::pane { /* The tab widget frame */
    border-top: 2px solid #C2C7CB;
    position: absolute;
    top: -0.5em;
}
QTabWidget::tab-bar {
    alignment: center;
}
QTabBar::tab {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
    border: 2px solid #C4C4C3;
    border-bottom-color: #C2C7CB; /* same as the pane color */
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
    min-width: 8ex;
    padding: 2px;
	background : qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1,
                                      stop: 0 white, stop: 0.4 white);
 
}</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Film">
     <attribute name="title">
      <string>Film</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>291</width>
        <height>401</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="title">
       <string>Ajouter film</string>
      </property>
      <widget class="QLabel" name="label_37">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>47</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_38">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_39">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>110</y>
         <width>47</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Genre</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_40">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>170</y>
         <width>47</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Durée</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_41">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>200</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Date de sortie</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_id">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>50</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
 
    background-color:QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #eef, stop: 1 #ccf);
padding: 1px;
border-style: solid;
border: 1px solid black;
border-radius: 4px;

}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_nom">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>80</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 1px solid grey;
border-radius: 4px;
}</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_genre">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>110</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 1px solid grey;
border-radius: 4px;
}</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit_ds">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>200</y>
         <width>161</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QDateEdit {
padding: 1px;
border-style: solid;
border: 1px solid grey;
border-radius: 4px;
background-color:QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #eef, stop: 1 #ccf);
}</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ajouter">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>360</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
  border: 1px solid blue;
  border-radius: 8px;
  padding: 0 8px;
background: lightgrey;
}
QPushButton:hover:!pressed{ color: blue}</string>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_update">
       <property name="geometry">
        <rect>
         <x>124</x>
         <y>360</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
  border: 1px solid orange;
  border-radius: 8px;
  background : lightgrey;
}
QPushButton:hover:!pressed{ color: orange}
font: 8pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Mettre à jour</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_42">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>140</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_descrip">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>140</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 1px solid grey;
border-radius: 4px;
}</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_duree">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>170</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 1px solid grey;
border-radius: 4px;
}</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_17">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>10</y>
        <width>561</width>
        <height>401</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="title">
       <string>Liste des films</string>
      </property>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>30</y>
         <width>261</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>8</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 1px solid grey;
border-radius: 4px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>30</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
  border: 1px solid grey;
  border-radius: 8px;
  background : lightgrey;
}
QPushButton:hover:!pressed{ color: purple}</string>
       </property>
       <property name="text">
        <string>Rechercher</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_pdfFilms">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>360</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
  border: 1px solid grey;
  border-radius: 8px;
  background : lightgrey;
}
</string>
       </property>
       <property name="text">
        <string>Exporter</string>
       </property>
      </widget>
      <widget class="QTableView" name="tableView_films">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>71</y>
         <width>511</width>
         <height>271</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QTableView QTableCornerButton::section {
    background: orange;
    border: 2px outset lightblue;
}
QTableView::indicator:unchecked {
    background-color: #d7d6d5
}</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_suppFilm">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>360</y>
         <width>75</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
  border: 1px solid red;
  border-radius: 8px;
  background : lightgrey;

}
QPushButton:hover:!pressed{ color: red}</string>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="Projection">
     <attribute name="title">
      <string>Projection</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_20">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>291</width>
        <height>401</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string>Ajouter projection</string>
      </property>
      <widget class="QLabel" name="label_82">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Date</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_83">
       <property name="geometry">
        <rect>
         <x>29</x>
         <y>100</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Heure</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_84">
       <property name="geometry">
        <rect>
         <x>29</x>
         <y>140</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>ID du film</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_idproj">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>140</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
padding: 1px;
border-style: solid;
border: 1px solid grey;
border-radius: 4px;
</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_update_proj">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>360</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
  border: 1px solid grey;
  border-radius: 8px;
  background : lightgrey;
}
font: 8pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Mettre à jour</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_85">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>180</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Numero de la salle</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_salle">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>180</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
padding: 1px;
border-style: solid;
border: 1px solid grey;
border-radius: 4px;
</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_ajouter_proj">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>360</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
  border: 1px solid green;
  border-radius: 8px;
  padding: 0 8px;
background: lightgrey;
}</string>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit_proj">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>60</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
    background-color:QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #eef, stop: 1 #ccf);
padding: 1px;
border-style: solid;
border: 1px solid grey;
border-radius: 4px;
</string>
       </property>
      </widget>
      <widget class="QTimeEdit" name="timeEdit_proj">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>100</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
    background-color:QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #eef, stop: 1 #ccf);
padding: 1px;
border-style: solid;
border: 1px solid grey;
border-radius: 4px;
</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_18">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>10</y>
        <width>491</width>
        <height>401</height>
       </rect>
      </property>
      <property name="title">
       <string>Liste des films</string>
      </property>
      <widget class="QLineEdit" name="lineEdit_rech">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>30</y>
         <width>261</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Yu Gothic UI Semilight</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">
padding: 1px;
border-style: solid;
border: 1px solid grey;
border-radius: 4px;
</string>
       </property>
       <property name="text">
        <string>date..</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_rechproj">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>30</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
  border: 1px solid grey;
  border-radius: 8px;
  background : lightgrey;
}
font: 8pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Rechercher</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_pdf_proj">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>360</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
  border: 1px solid grey;
  border-radius: 8px;
  background : lightgrey;
}
font: 8pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Exporter</string>
       </property>
      </widget>
      <widget class="QTableView" name="tableView_projections">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>71</y>
         <width>461</width>
         <height>271</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_supp_proj">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>360</y>
         <width>75</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
  border: 1px solid red;
  border-radius: 8px;
  background : lightgrey;
}
font: 8pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Stats</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>968</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
